wallet/helpers/parseMnemonicOrHdNodeString.js

```diff
@@ -0,0 +1,19 @@
+const CardanoCrypto = require('cardano-crypto.js')
+
+const {isPaperWalletMnemonic, decodePaperWalletMnemonic} = require('../mnemonic')
+
+module.exports = async (mnemonicOrHdNodeString) => {
```

Comment:
I would suggest `const parseMnemonicOrHdNodeString =` and then at the bottom of file `module.exports = parseMnemonicOrHdNodeString`

This way you will be able search and find the function in the code